src/components/Screens/Auth/CustomerDashboard.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable class-methods-use-this */
+import React, { Component } from 'react';
+import Grid from '@material-ui/core';
+
+class CustomerDashboard extends Component {
+  render() {
+    return (
+      <Grid>
+        <Grid>
+          <Typography>
+            <Link href="#" onClick={preventDefault}>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['preventDefault' is not defined.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5592511)